openhtf/conf.py

```diff
@@ -257,29 +256,31 @@ def Load(self, config_file=None, force_reload=False,
       return False
 
     try:
-      filename = config_file or FLAGS.config
-      _LOG.info('Loading from config: %s', filename)
-
-      with config_loader(filename) as config_file:
-        data = yaml.safe_load(config_file)
-        if not data:
-          raise ConfigurationInvalidError('No data', config_file)
-        self._state.clear()
-        self._state.update(data)
+      if FLAGS.config:
+        _LOG.info('Loading from config: %s', FLAGS.config)
+        with config_loader(FLAGS.config) as config_file:
+          data = yaml.safe_load(config_file)
+          if not data:
+            raise ConfigurationInvalidError('No data', config_file)
+          self._state.clear()
+          self._state.update(data)
 
       # Load string values from flags
       for keyval in FLAGS.config_value:
         key, val = keyval.split('=')
         self._state[key] = val
+   
+      if 'station_id' not in self._state:
```

Comment:
If station_id key isn't present we see the following:

Traceback (most recent call last):
  File "./hello_world.py", line 91, in <module>
    test.Execute(test_start=triggers.PromptForTestStart())
  File "/usr/local/google/home/wallaceb/openhtf/openhtf/__init__.py", line 204, in Execute
    os.getpid()).SaveToFile(FLAGS.rundir)
  File "/usr/local//home/openhtf/openhtf/io/rundata.py", line 71, in SaveToFile
    filename = os.path.join(directory, self.station_id)
  File "/usr/local//home/venv/lib/python2.7/posixpath.py", line 75, in join
    if b.startswith('/'):
AttributeError: 'NoneType' object has no attribute 'startswith'